test/functional/legacy/003_cindent_spec.lua

```diff
@@ -1832,6 +1852,26 @@ describe('cindent', function()
       {
       	111111111111111111;
       }
+      namespace test::cpp17
+      {
+      	111111111111111111;
+      }
+      namespace ::incorrectcpp17
+      {
+      	111111111111111111;
```

Comment:
In Vim's test the expected result has a tab:

    	111111111111111111;

the other tests look different too...